app/pubspec.lock

```diff
@@ -266,4 +280,4 @@ packages:
     version: "2.1.2"
 sdks:
   dart: ">=2.10.0-110 <2.11.0"
-  flutter: ">=1.12.13+hotfix.5 <2.0.0"
```

Comment:
Did `flutter pub get` update this?

Reply:
Most likely, flutter pub get also updates dependencies